### DISCUSSION TITLE
Capture video frames with orientation information

### DISCUSSION THREAD

**Discussion by viswavsn81** [opentrack/opentrack#1750] [Q&A]
Hello, I need to capture a video of a subject and tag each frame with the computed orientation information. Could you give some guidance on how this can be done ? Thanks

**Comment by sthalik** [comment]
Sure. Use `cv::VideoWriter` https://docs.opencv.org/4.x/dd/d9e/classcv_1_1VideoWriter.html. But you haven't said anything about what the  video frames consist of, so it's not a complete solution.

**Comment by viswavsn81** [comment]
Thank you for your response. I am using the camera to track the head of a person. I need to store the individual frames of the video capture and tag the orientation information of that specific  frame in a table format ( database ). 

Bing chat provided these steps - The work class emits signals when the camera image or the tracker data changes, and the video widget connects to these signals to update its content. The tracker data consists of the yaw, pitch, roll, x, y, and z values of the head movement, which are represented by a dinput::vec6 object. 

I suppose I must use this signal from the work class to trigger the frame capture . Is this approach right ?

**Reply by sthalik** [reply]
Just get a programmer.
